content/documentation/pages/2-concepts/3-streams.md

```diff
@@ -6,11 +6,14 @@ description: 'Stream Processing Framework and Concepts'
 
 # Stream Processing
 
-Stream processing, is defined as the processing of infinite amount of data without interaction or interruption.
-Use cases for stream processing are real-time credit card fraud detection or predictive analytics.
-
-Stream processing in Data Flow is implemented architecturally as a collection of independent event-driven streaming applications that connect using messaging middleware, for example RabbitMQ or Apache Kafka.
-The collection of application is referred to as a streaming data pipelines.
+Stream processing is defined as the processing of an unbounded amount of data without interaction or interruption.
+Business cases for stream processing include:
+- Real-time credit card fraud detection or predictive analytics.
+- Continuous integration with machine learning platforms
+- Near Real-time busines data processing for actionable analytics
+
+Stream processing in Spring Cloud Data Flow is implemented architecturally as a collection of independent event-driven streaming applications that connect using a messaging middleware of choice, for example RabbitMQ or Apache Kafka.
+The collection of applications that comprise a workflow is referred to as a streaming data pipeline.
```

Comment:
"workflow" doesn't seem to align well with the message. Maybe:

> The collection of independent applications come together at runtime to constitute a streaming data pipeline.

Reply:
I Like that a lot better.